<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CVE_Dialog</class>
 <widget class="QDialog" name="CVE_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>454</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>626</width>
    <height>454</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>626</width>
    <height>454</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Upload CVE</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog { background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(17, 5, 0, 255), stop:0.081448 rgba(66, 46, 24, 255), stop:0.104072 rgba(30, 26, 73, 255), stop:0.41629 rgba(7, 4, 37, 255), stop:0.714932 rgba(126, 14, 81, 255), stop:0.751131 rgba(29, 13, 13, 255), stop:0.886878 rgba(70, 70, 51, 255)); }</string>
  </property>
  <widget class="QDialogButtonBox" name="CVE_buttonBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>410</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>626</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="Platform_groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>611</width>
     <height>401</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ubuntu-light</family>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="statusTip">
    <string>Platform</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox { font-family: ubuntu-light; border: 2px solid gray;  border-radius: 5px; margin-top: 1ex; background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(228, 251, 79, 206), stop:0.4 rgba(244, 252, 255, 241), stop:0.425 rgba(244, 252, 255, 241), stop:0.44 rgba(244, 252, 255, 241), stop:1 rgba(244, 252, 255, 241)); }</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="title">
    <string>CVE</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>241</width>
      <height>34</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="CVE_Platform_Label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>CVE Platform:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="CVE_Platform_Edit">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>251</width>
      <height>34</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="CVE_Platform_Ver_Label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>CVE Email Author:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="CVE_Author_Edit">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="CVE_Description_Label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>170</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>CVE Description</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="CVE_Description_Text">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>581</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>541</width>
      <height>34</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="CVE_Platform_Ver_Label_2">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>CVE Documentation:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="CVE_Document_Edit">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CVE_pdf">
       <property name="text">
        <string>Documentation</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>241</width>
      <height>34</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="CVE_Platform_Version_Label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>CVE Platform Version:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="CVE_Platform_Version_Edit">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>50</y>
      <width>251</width>
      <height>34</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="CVE_Platform_Ver_Label_3">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>CVE Title:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="CVE_Title_Edit">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>CVE_buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CVE_Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CVE_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CVE_Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
